buildenv/jenkins/common/build.groovy

```diff
@@ -565,36 +565,51 @@ def add_node_to_description() {
     currentBuild.description = TMP_DESC + "<a href=${JENKINS_URL}computer/${NODE_NAME}>${NODE_NAME}</a>"
 }
 
+def cleanWorkspace (KEEP_WORKSPACE) {
+        if (KEEP_WORKSPACE) {
+		try {	
+			cleanWs disableDeferredWipeout: true, deleteDirs: true
+		} catch (Exception e) {
+			try {
+				if (SPEC.contains('win')) {
+					WORKSPACE = sh(script: "cygpath -u '${env.WORKSPACE}'", returnStdout: true).trim()
+				}
+				if (WORKSPACE) {
+					sh "rm -fr ${WORKSPACE}/*"
+				}
+			} catch(Exception f) {
+				println("Exception: ${f}")
+			}
+		}
+	}
+}
+
 def build_all() {
-    stage ('Queue') {
-        timeout(time: 10, unit: 'HOURS') {
-            node("${NODE}") {
-                timeout(time: 5, unit: 'HOURS') {
-                    try {
-                        // Cleanup in case an old build left anything behind
-                        cleanWs disableDeferredWipeout: true, deleteDirs: true
-                        add_node_to_description()
-                        // initialize BOOT_JDK, FREEMARKER, OPENJDK_REFERENCE_REPO here
-                        // to correctly expand $HOME variable
-                        variableFile.set_build_variables_per_node()
-                        get_source()
-                        variableFile.set_sdk_variables()
-                        variableFile.set_artifactory_config(BUILD_IDENTIFIER)
-                        variableFile.set_build_custom_options()
-                        build()
-                        archive_sdk()
-                    } finally {
-                        KEEP_WORKSPACE = (params.KEEP_WORKSPACE) ?: false
-                        if (!KEEP_WORKSPACE) {
-                            // disableDeferredWipeout also requires deleteDirs. See https://issues.jenkins-ci.org/browse/JENKINS-54225
-                            cleanWs notFailBuild: true, disableDeferredWipeout: true, deleteDirs: true
-                        }
-                    }
-                }
-            }
-            upload_artifactory_post()
-        }
-    }
+	stage ('Queue') {
+		timeout(time: 10, unit: 'HOURS') {
+			node("${NODE}") {
+				timeout(time: 5, unit: 'HOURS') {
+					try {
+						cleanWorkspace("false")
+						add_node_to_description()
+						// initialize BOOT_JDK, FREEMARKER, OPENJDK_REFERENCE_REPO here
+						// to correctly expand $HOME variable
+						variableFile.set_build_variables_per_node()
+						get_source()
+						variableFile.set_sdk_variables()
+						variableFile.set_artifactory_config(BUILD_IDENTIFIER)
+						variableFile.set_build_custom_options()
+						build()
+						archive_sdk()
+					} finally {
+						KEEP_WORKSPACE = (params.KEEP_WORKSPACE) ?: false
+						cleanWorkspace(KEEP_WORKSPACE)
+					}
+				}
+			}
+			upload_artifactory_post()
+		}
+	}
```

Comment:
Looks like you've changed the spaces to tabs which is why we get the big whitespace diff.